Libraries/ART/ARTNode.m

```diff
@@ -14,7 +14,16 @@ @implementation ARTNode
 - (void)insertReactSubview:(UIView *)subview atIndex:(NSInteger)atIndex
 {
   [super insertReactSubview:subview atIndex:atIndex];
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+  NSArray<__kindof NSView *> *subviews = self.subviews;
```

Comment:
This is done a few times.   We should make a static line in RCTUIKit.h like RCTUIViewInsertSubView()